### PR TITLE
Stereocam2state pullrequest kim

### PR DESCRIPTION
**Pull request by knmcguire**
-> GPS from abi message
-> Unused functions removed
-> Fixed rewrite of stereocam msg for state measurements

### CODE REVIEW

File: sw/airborne/modules/stereocam/stereocam2state/stereocam2state.c [hunk 1]
```diff
@@ -34,13 +39,29 @@
...
+  gps_stereocam.ecef_vel.x = gps_s->ecef_vel.x;
+  gps_stereocam.ecef_vel.y = gps_s->ecef_vel.y;
+  gps_stereocam.ecef_vel.z = gps_s->ecef_vel.y;
+
+}
+
+void stereo_to_state_init(void)
+{
+  //subscribe to GPS abi-messages for state measurements
+  AbiBindMsgGPS(ABI_BROADCAST, &gps_ev, stereocam_gps_cb);
```
Comment (flixr) [inline sw/airborne/modules/stereocam/stereocam2state/stereocam2state.c]: I guess you wanted to use `STEREOCAM_GPS_ID` instead of `ABI_BROADCAST` here...

**Comment by flixr** [comment]
Nice, that was quick :-)

File: sw/airborne/modules/stereocam/stereocam2state/stereocam2state.c (hunk 1 above)
Comment (knmcguire) [inline sw/airborne/modules/stereocam/stereocam2state/stereocam2state.c]: Woops yes that was indeed the intention, let me just recommit this again

**Comment by flixr** [comment]
Do you mind if I squash them all together into one commit before merging?

**Comment by knmcguire** [comment]
I don't mind, probably better for the overview. Beware that that the stereocam2state.xml before commit 3c03dfa was not compatible with the masterbranch. Should not make a difference if you squash them together?

**Pull request closed without merging**